sixtools/AD_Handler.py

```diff
@@ -0,0 +1,60 @@
+from databroker.assets.handlers_base import HandlerBase
+import os.path
+import pandas as pd
+import numpy as np
+
+
+class AreaDetector_HDF5SingleHandler_DataFrame(HandlerBase):
+    '''Handler for hdf5 data stored 1 image per file and returned as a
+    Pandas.DataFrame.
+
+    This will work with all hdf5 files that are a mxn arrays and the data is
+    'table like' where m is the number of columns and n is the number of rows.
+
+    Parameters
+    ----------
+    fpath : string
+        filepath
+    template : string
+        filename template string.
+    filename : string
+        filename
+    key : string
+        the 'path' inside the file to the data set.
+    frame_per_point : float
+        the number of frames per point.
+    '''
+    specs = {'AD_HDF5_SINGLE_DATAFRAME'} | HandlerBase.specs
```

Comment:
This does not match the spec in https://github.com/NSLS-II-SIX/profile_collection/pull/17. Once you settle on a name there, whether it's `AD_HDF5_XIP` or `AD_HDF5_SIN`GLE_XIP`, it should match here.

Sidebar: As a point of interest, the handler would actually still be _usable_ even if it didn't have the spec we wanted here. One can always `register_handler('WHATEVER_I_WANT', MyHandler)`. The `specs` attribute just makes auto-registration easy to implement, and it's a form of machine-readable documentation.

Reply:
I purposely did not have them match, as per the discussion in the group meeting. 

This name relates to a 'generic' name which can relate to any use of this handler, the one in the SIX profile_collection is specifically related to the XIP plugin for that detector. I have made association between the name used in the six profile and this handler in the six playbooks.

Reply:
I see. :+1: